l2geth/internal/ethapi/api.go

```diff
@@ -54,7 +54,7 @@ const (
 )
 
 var errOVMUnsupported = errors.New("OVM: Unsupported RPC Method")
-var bigDefaultGasPrice = new(big.Int).SetUint64(defaultGasPrice)
+var bigDefaultGasPrice = new(big.Int).SetUint64(0) // TEMPORARY
```

Comment:
This could use a better comment as to why its temporary (making tests pass bc no message passing)